pkg/sac/scope_checker_or.go

```diff
@@ -0,0 +1,212 @@
+package sac
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/sac/effectiveaccessscope"
+)
+
+type orScopeChecker struct {
+	scopeCheckers []ScopeChecker
+}
+
+// NewOrScopeChecker returns a new scope checker wrapping the given scope checkers,
+// each result will be combined in a disjunction (OR).
+func NewOrScopeChecker(scopeCheckers ...ScopeChecker) ScopeChecker {
+	return orScopeChecker{
+		scopeCheckers: scopeCheckers,
+	}
+}
+
+func (s orScopeChecker) Core() ScopeCheckerCore {
+	if len(s.scopeCheckers) != 0 {
+		return s.scopeCheckers[0].Core()
```

Comment:
Why do we select the first one?

Reply:
Essentially discussed this within https://github.com/stackrox/stackrox/pull/1492#discussion_r861402923, bottom-line is that I don't really see an unambiguous way to return a ScopeCheckerCore, hence the first one.